demo/psa/automated-end-to-end.md

```diff
@@ -9,8 +9,11 @@ Please ensure:
 
 ## Setup and running of docker containers
 
+NOTE: The environment variable `GIT_CLONE_BRANCH` specified in `default.env` in the `veraison/services` repo should be set to `demo-psa-1.0.1` when running this end to end demonstration (by default `GIT_CLONE_BRANCH` is set to `main`).
+
```

Comment:
that's good, maybe even better if this could be automated in some way, e.g.:
* using `-e GIT_CLONE_BRANCH=demo-psa-1.0.1` on the command line
* adding a demo.env file with the right setting for the `GIT_CLONE_BRANCH` variable

Reply:
Yes I could do this, Ill open a new PR for this change in veraison/services